car-bridge/README.md

```diff
@@ -0,0 +1,8 @@
+# Car Bridge
+
+The Car Bridge is a component that integrates Chariott with the cloud.
+
+To run the Car Bridge:
+
+1. Start an [Eclipse Mosquitto](https://mosquitto.org/) broker
+2. Start the Car Bridge using `cargo run -p car-bridge`
```

Comment:
nit:

```suggestion
1. Start an [Eclipse Mosquitto](https://mosquitto.org/) broker.

2. Start the Car Bridge using:

       cargo run -p car-bridge
```

Make executable instructions copy-paste friendly rather than embedding them within sentences.